pandas/tests/indexes/datetimes/test_timezones.py

```diff
@@ -434,24 +434,30 @@ def test_dti_tz_localize_utc_conversion(self, tz):
         with pytest.raises(pytz.NonExistentTimeError):
             rng.tz_localize(tz)
 
+    @pytest.mark.parametrize('tz', [
```

Comment:
We shouldn't need this; just keep the original `tz_aware_fixture`

Reply:
done. i've picked a range which hopefully doesnt have a DST transition in any timzeone, but i havent exhaustively checked that is true.